docs/dev/src/build-and-run/intro.md

```diff
@@ -7,7 +7,7 @@
 To install the dependencies on macOS, run:
 
 ```shell
-brew install postgresql cmake protobuf tmux cyrus-sasl lld openssl@3
+brew install postgresql cmake protobuf tmux cyrus-sasl lld openssl@3 libomp
```

Comment:
I also tried `brew uninstall libomp` and our fork with `PATH` pointing to brew-llvm can still compile. Maybe it is unnecessary.

We can also keep it here but encourage people to try building without this extra install first, and remove it when we are confident.